data/tooltips.txt

```diff
@@ -466,3 +466,6 @@ tip "solar power?"
 
 tip "no hyperdrive?"
 	`This ship has no hyperdrive. If it took off, it would be unable to leave this star system.`
+
+tip "no fuel?"
+	`This ship has not enough fuel capacity to make a hyperspace jump. If it took off, it would be unable to leave this star system.`
```

Comment:
"has not enough fuel" isn't grammatically correct in English. It needs to be "doesn't have enough fuel" or "has insufficient fuel" or something like that instead.

Reply:
Actually "has not enough" *is* grammatically correct in English. Maybe it's less common in American, but that's a matter of idiom, not grammar.